RNIap.podspec

```diff
@@ -4,14 +4,13 @@ package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
 Pod::Spec.new do |s|
   s.name         = "RNIap"
   s.version      = package['version']
-  s.summary      = "RNIap"
-  s.description  = package['description']
+  s.summary	 = package['description']
   s.homepage     = "https://github.com/dooboolab/react-native-iap"
   s.license      = "MIT"
   s.author       = package['author']
-  s.platform     = :ios, "7.0"
-  s.source       = { :git => "https://github.com/dooboolab/react-native-iap", :tag => "v#{s.version}" }
-  s.source_files  = "ios/**/*.{h,m}"
+  s.platform     = :ios, "9.0"
+  s.source       = { :git => "https://github.com/dooboolab/react-native-iap", :tag => "#{s.version}" }
+  s.source_files  = "ios/*.{h,m}"
```

Comment:
Could you also explain how this could fix your problem?

Reply:
It is related auto linking
current podspec source_files path is "ios/**/*.{h,m}", but there is no files to link
```
ios/RNIapIos.m
ios/RNIapIos.h
ios/IAPPromotionObserver.h
ios/IAPPromotionObserver.m
= ios/*.{h,m}
```

Reply:
Thanks again for this! Looks quite reasonable and I'll merge it for now.

Reply:
thanks for confirming!